editor/templates/place/edit.html

```diff
@@ -212,6 +262,64 @@ <h3>Editing {{ wof_doc.properties['wof:name'] or wof_doc.id }}</h3><small>(<a hr
 </div>
 
 <form class="needs-validation" novalidate action="{{ url_for('place.edit_place', url=wof_url) }}" method="POST">
+    <h4>Geometries</h4>
+    <!-- https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/wof/name.json -->
+    <div class="form-group">
+        {%- if wof_doc.properties['geom:area_square_m'] is defined %}
+        <small id="wof-name-help" class="form-text text-muted">Land-only multi-polygons are preferred, points are allowed. {{ wof_doc.properties['geom:area_square_m'] }} square meters</small>
```

Comment:
To match the sequencing in the Common properties section of the form, the small helper text should appear _after_ the input (not above)... but the CSS would need a variant added else there's too much spacing added _above_ making it look like the text is for the next input not the current input.

(Applies to all in the geometries section)